.github/workflows/build-and-install.yml

```diff
@@ -124,3 +124,80 @@ jobs:
+          # - 'centos:8'
+          - 'debian:buster'
+          - 'fedora:32'
+          - 'ubuntu:20.04'
+        include:
+          #- distro: 'centos:8'
+          #  pre: >-
+          #    yum -y update &&
+          #    yum -y groupinstall 'Development Tools' &&
+          #    yum -y install libcurl-devel openssl-devel libuuid-devel
+          #  build_kinesis: >-
+          #    git clone https://github.com/aws/aws-sdk-cpp.git &&
+          #    cmake -DCMAKE_INSTALL_LIBDIR=/usr/lib
+          #    -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
+          #    -DBUILD_SHARED_LIBS=OFF -DBUILD_ONLY=kinesis
+          #    ./aws-sdk-cpp
+          - distro: 'debian:buster'
+            pre: >-
+              apt-get update &&
+              apt-get install -y build-essential &&
+              apt-get install -y libcurl4-openssl-dev libssl-dev uuid-dev zlib1g-dev libpulse-dev
+            build_kinesis: >-
+              git clone https://github.com/aws/aws-sdk-cpp.git &&
+              cmake -DCMAKE_INSTALL_LIBDIR=/usr/lib
+              -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
+              -DBUILD_SHARED_LIBS=OFF -DBUILD_ONLY=kinesis
+              ./aws-sdk-cpp
+          - distro: 'fedora:32'
+            pre: >-
+              dnf -y update &&
+              dnf -y groupinstall 'Development Tools' &&
+              dnf -y install libcurl-devel openssl-devel libuuid-devel
+            build_kinesis: >-
+              git clone https://github.com/aws/aws-sdk-cpp.git &&
+              cmake -DCMAKE_INSTALL_LIBDIR=/usr/lib
+              -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
+              -DBUILD_SHARED_LIBS=OFF -DBUILD_ONLY=kinesis
+              ./aws-sdk-cpp
+          - distro: 'ubuntu:20.04'
+            pre: >-
+              apt-get update &&
+              apt-get install -y build-essential &&
+              apt-get install -y libcurl4-openssl-dev libssl-dev uuid-dev zlib1g-dev libpulse-dev
+            build_kinesis: >-
+              git clone https://github.com/aws/aws-sdk-cpp.git &&
+              cmake -DCMAKE_INSTALL_LIBDIR=/usr/lib
+              -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
+              -DBUILD_SHARED_LIBS=OFF -DBUILD_ONLY=kinesis
+              ./aws-sdk-cpp
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git clone repository
+        uses: actions/checkout@v2
+      - name: install-required-packages.sh on ${{ matrix.distro }}
+        env:
+          PRE: ${{ matrix.pre }}
+          BUILD_KINESIS: ${{ matrix.build_kinesis }}
+        run: |
+          echo $PRE > ./prep-cmd.sh
+          echo $BUILD_KINESIS > ./build-kinesis.sh && chmod +x ./build-kinesis.sh
+          docker build . -f .github/dockerfiles/Dockerfile.build_test -t test --build-arg BASE=${{ matrix.distro }}
+      - name: Build on ${{ matrix.distro }}
+        env:
+          RUNCMD: >-
+            ./build-kinesis.sh &&
+            ./netdata-installer.sh \--dont-wait --dont-start-it \
+            --enable-backend-kinesis
```

Comment:
@prologic it doesn't build Netdata. Only AWS SDK is built.

Reply:
Not sure sure I'm following?

```#!console
./netdata-installer.sh \--dont-wait --dont-start-it \
            --enable-backend-kinesis
```

Isn't this doing what we want here?

Reply:
It doesn't run this part:
```
            ./netdata-installer.sh \--dont-wait --dont-start-it \
            --enable-backend-kinesis
```

Reply:
Look at https://github.com/netdata/netdata/pull/9823/checks?check_run_id=1079824682#step:4:3852
There are no log messages for Netdata build after AWS SDK has been installed.

Reply:
Hmmmm